widget/embedded/src/containers/Wallets/Wallets.tsx

```diff
@@ -120,7 +122,7 @@ function Main(props: PropsWithChildren<PropTypes>) {
         allBlockChains={blockchains}
         providers={providers}
         onUpdateState={onUpdateState}
-        autoConnect>
+        autoConnect={!!isActiveTab}>
```

Comment:
Do we need this? because autoConnect will silently connect wallets. After activating tab, user should manually connect them again.

Reply:
Auto-connect is triggered only once when its value becomes true.

Reply:
We talked offline, keeping this is more safe.